### PR TITLE
K8SPG-617: Add PG operator 2.4.1

### PR DESCRIPTION
**Pull request by jvpasinatto**

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

**Review by egegunes — APPROVED** [review]

File: sources/operator.2.4.1.pg-operator.json [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+            "critical": false
+          },
+          "12.17": {
+            "image_path": "percona/percona-postgresql-operator:2.3.1-ppg12-postgres",
+            "image_hash": "f65fec42a82c937ea0ce1b25898a245d469375ed6ffddd2a396a032c86ccc2ee",
+            "status": "available",
+            "critical": false
+          },
+
+          "13.15": {
```
Comment (ptankov) [inline sources/operator.2.4.1.pg-operator.json:21]: there is no such tag `2.4.1-ppg13.15-postgres`

File: sources/operator.2.4.1.pg-operator.json [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+          },
+
+          "15.7": {
+            "image_path": "percona/percona-postgresql-operator:2.4.1-ppg15.7-postgres",
+            "image_hash": "8d58c8e9d7c54849854027c36905c149fa1db5ea4e8d13b3e6cb69d6e8128c7f",
+            "status": "recommended",
+            "critical": false
+          },
+          "15.5": {
+            "image_path": "percona/percona-postgresql-operator:2.3.1-ppg15-postgres",
```
Comment (ptankov) [inline sources/operator.2.4.1.pg-operator.json:54]: there is no such tag `2.3.1-ppg15-postgres`

File: sources/operator.2.4.1.pg-operator.json [hunk 3]
```diff
@@ -0,0 +1,291 @@
...
+          },
+
+          "16.3": {
+            "image_path": "percona/percona-postgresql-operator:2.4.1-ppg16.3-postgres",
+            "image_hash": "8248b290a88b881f1871fbca0de7da1acace31f94f795d1990e3ca3ca5dd3636",
+            "status": "recommended",
+            "critical": false
+          },
+          "16.1": {
+            "image_path": "percona/percona-postgresql-operator:2.3.1-ppg16-postgres",
```
Comment (ptankov) [inline sources/operator.2.4.1.pg-operator.json:67]: there is no such tag `2.3.1-ppg16-postgres`

**Comment by ptankov** [comment]
check all 2.3.1 tags

**Pull request merged**